src/Util/TypeWriter.lua

```diff
@@ -0,0 +1,17 @@
+return function(label: TextLabel, text: string, delayBetweenChars: number?)
+	local displayText = text
+
+	-- Replace line break tags so grapheme loop will not miss those characters
+	displayText = displayText:gsub("<br%s*/>", "\n")
+	displayText:gsub("<[^<>]->", "")
```

Comment:
Do we use `String:gsub(...)` in other parts of the project as opposed to `string.gsub(String, ...)`?
Behavior-wise, I'm not sure this properly handled mismatched tags (`<a>thing</b>` -> `thing`, when it shouldn't). Handling that may be tricky. It wasn't easy to get right in Nexus Admin for string escaping (https://github.com/TheNexusAvenger/Nexus-Admin/blob/master/src/Common/Filter.lua#L29-L42).

Reply:
Wil look into handling mismatched tags.